test/projections.jl

```diff
@@ -76,3 +77,56 @@ end
     output_joint = MOD.projection_gradient_on_set(DD, [v1, v2], [c1, c2])
     @test output_joint ≈ BlockDiagonal([output_1, output_2])
 end
+
+
+@testset "Exponential Cone Projections" begin
+    function det_case_exp_cone(v; dual=false)
+        v = dual ? -v : v
+        if MOD.distance_to_set(DD, v, MOI.ExponentialCone()) < 1e-8
+            return 1
+        elseif MOD.distance_to_set(DD, -v, MOI.DualExponentialCone()) < 1e-8
+            return 2
+        elseif v[1] <= 0 && v[2] <= 0 #TODO: threshold here??
+            return 3
+        else
+            return 4
+        end
+    end
+
+    function _test_proj_exp_cone_help(x, tol; dual=false)
+        cone = dual ? MOI.DualExponentialCone() : MOI.ExponentialCone()
+        model = Model()
+        set_optimizer(model, optimizer_with_attributes(
+            SCS.Optimizer, "eps" => 1e-10, "max_iters" => 10000, "verbose" => 0))
+        @variable(model, z[1:3])
+        @variable(model, t)
+        @objective(model, Min, t)
+        @constraint(model, sum((x-z).^2) <= t)
```

Comment:
Random note:
interesting, I didn't think this way of writing the squared norm would be faster than dot:
```
julia> @btime @constraint($model, sum(($x-$z).^2) <= $t)
  4.522 μs (112 allocations: 9.39 KiB)
julia> @btime @constraint($model, ($x-$z) ⋅ ($x-$z)  <= $t)
  6.419 μs (168 allocations: 13.28 KiB)
```

Reply:
Oh interesting. Admittedly, I didn't think to check the latter.